src/containers/pages/MDAPoint/plans/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -139,164 +139,237 @@ exports[`components/MDA Reports/MDAPlansList renders plan definition list correc
+      onClick={[Function]}
+      onMouseDown={[Function]}
+      onTouchStart={[Function]}
+      role="separator"
+      style={
+        Object {
+          "cursor": "ew-resize",
+        }
+      }
+    />
+  </div>
+  <div
+    className="th"
+    colSpan={1}
+    key="thead-th-3"
+    onClick={[Function]}
+    role="columnheader"
+    style={
+      Object {
+        "boxSizing": "border-box",
+        "cursor": "pointer",
+        "flex": "150 0 auto",
+        "minWidth": "50px",
+        "position": "relative",
+        "width": "150px",
+      }
+    }
+    title="Toggle SortBy"
   >
-    retired
-  </td>,
-  <td
-    className="listview-th"
-    key="0"
-  >
-    <Link
-      key="9f19b77c-b9a5-5832-a4e5-4b461d18fce7"
-      replace={false}
-      to="/intervention/mda-point/report/9f19b77c-b9a5-5832-a4e5-4b461d18fce7"
+    End Date
+    <SortIcon
+      isSorted={false}
     >
-      <a
-        href="/intervention/mda-point/report/9f19b77c-b9a5-5832-a4e5-4b461d18fce7"
-        onClick={[Function]}
+      <div
+        className="icon-wrapper"
       >
-        MDA-Point-2019-08-30m
-      </a>
-    </Link>
-  </td>,
-  <td
-    className="listview-th"
-    key="1"
-  >
-    2019-08-30
-  </td>,
-  <td
-    className="listview-th"
-    key="2"
-  >
-    2019-08-30
-  </td>,
-  <td
-    className="listview-th"
-    key="3"
-  >
-    2019-09-14
-  </td>,
-  <td
-    className="listview-th"
-    key="4"
-  >
-    active
-  </td>,
-  <td
-    className="listview-th"
-    key="0"
+        <div
+          className="previous-next-filled  icon"
+        />
+      </div>
+    </SortIcon>
+    <div
+      className="resizer "
+      draggable={false}
+      onClick={[Function]}
+      onMouseDown={[Function]}
+      onTouchStart={[Function]}
+      role="separator"
+      style={
+        Object {
+          "cursor": "ew-resize",
+        }
+      }
+    />
+  </div>
+  <div
+    className="th"
+    colSpan={1}
+    key="thead-th-4"
+    onClick={[Function]}
+    role="columnheader"
+    style={
+      Object {
+        "boxSizing": "border-box",
+        "cursor": "pointer",
+        "flex": "150 0 auto",
+        "minWidth": "50px",
+        "position": "relative",
+        "width": "150px",
+      }
+    }
+    title="Toggle SortBy"
   >
-    <Link
-      key="c6e59ec1-8c54-5fdf-acf3-ee3692983f38"
-      replace={false}
-      to="/intervention/mda-point/report/c6e59ec1-8c54-5fdf-acf3-ee3692983f38"
+    Status
+    <SortIcon
+      isSorted={false}
     >
-      <a
-        href="/intervention/mda-point/report/c6e59ec1-8c54-5fdf-acf3-ee3692983f38"
-        onClick={[Function]}
+      <div
+        className="icon-wrapper"
       >
-        Berg Eswatini 2019
-      </a>
-    </Link>
-  </td>,
-  <td
-    className="listview-th"
-    key="1"
-  >
-    2019-08-29
-  </td>,
-  <td
-    className="listview-th"
-    key="2"
-  >
-    2019-08-29
-  </td>,
-  <td
-    className="listview-th"
-    key="3"
-  >
-    2019-09-18
-  </td>,
```

Comment:
I am nervous here.  We are removing the data in the row from the snapshot.  Why?

Reply:
The snapshot is too big i decided to just check the number of  items being rendered.